cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/store/NamespaceSourceControlMetadataStoreTest.java

```diff
@@ -33,29 +45,269 @@ public abstract class NamespaceSourceControlMetadataStoreTest {
   private static final String SPEC_HASH = "testSpecHash";
   private static final Instant LAST_MODIFIED = Instant.now();
   private static final String NAME = "testName";
+  private static final Boolean IS_SYNCED = true;
+  private static final String TYPE = EntityType.APPLICATION.toString();
   private static final ApplicationReference APP_REF = new ApplicationReference(NAMESPACE, NAME);
   private static final SourceControlMeta SOURCE_CONTROL_META = new SourceControlMeta(SPEC_HASH,
-      COMMIT_ID, LAST_MODIFIED, true);
+      COMMIT_ID, LAST_MODIFIED, IS_SYNCED);
 
   protected static TransactionRunner transactionRunner;
 
   @Before
   public void before() {
     TransactionRunners.run(transactionRunner, context -> {
-      NamespaceSourceControlMetadataStore store = NamespaceSourceControlMetadataStore.create(context);
+      NamespaceSourceControlMetadataStore store = NamespaceSourceControlMetadataStore.create(
```

Comment:
Please revert back all reformatting that is not needed

Reply:
sure, done